shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -792,16 +792,19 @@ - (NSUInteger)supportedInterfaceOrientations {
 #pragma mark - Accessibility
 
 - (void)onAccessibilityStatusChanged:(NSNotification*)notification {
+  auto platformView =  _shell->GetPlatformView();
 #if TARGET_OS_SIMULATOR
   // There doesn't appear to be any way to determine whether the accessibility
   // inspector is enabled on the simulator. We conservatively always turn on the
-  // accessibility bridge in the simulator.
-  bool enabled = true;
+  // accessibility bridge in the simulator, but never assistive technology.
+  platformView->SetSemanticsEnabled(true);
+  platformView->SetAssistiveTechnologyEnabled(false);
 #else
   bool enabled = UIAccessibilityIsVoiceOverRunning() || UIAccessibilityIsSwitchControlRunning() ||
-                 UIAccessibilityIsSpeakScreenEnabled();
+                UIAccessibilityIsSpeakScreenEnabled();
```

Comment:
Should "SpeakScreen" set assitiveTechnologiesEnabled to true? This one feels kinda borderline...

Reply:
Fixed?